x-pack/plugins/stack_alerts/public/rule_types/es_query/util.ts

```diff
@@ -41,4 +42,21 @@ export const convertFieldSpecToFieldOption = (fieldSpec: FieldSpec[]): FieldOpti
     });
 };
 
+export const convertRawRuntimeFieldtoFieldOption = (
+  rawFields: estypes.MappingRuntimeFields
+): FieldOption[] => {
+  const result: FieldOption[] = [];
+  for (const name of Object.keys(rawFields)) {
+    const rawField = rawFields[name];
+    const type = rawField.type;
+
+    const normalizedType = NORMALIZED_FIELD_TYPES[type] || type;
+    const aggregatable = true;
+    const searchable = true;
```

Comment:
I am not sure if all the possible runtime fields are always searchable/aggregatable: https://www.elastic.co/guide/en/elasticsearch/reference/current/runtime-mapping-fields.html

For example, `lookup` and `composite`? Are those searchable/aggregatable?

Reply:
Resolved in this commit [583cd97](https://github.com/elastic/kibana/pull/160319/commits/583cd9762ba33b4b7c079a871b3117400d1c6209)

Reply:
New issue https://github.com/elastic/kibana/issues/161595